packages/help-center/src/data/use-submit-a4a-support-ticket.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-restricted-imports */
+import { useMutation } from '@tanstack/react-query';
+import wpcom from 'calypso/lib/wp';
```

Comment:
Typically, we would use `wpcomRequest` here, but for the A4A endpoint, there are additional cookies that are only inherited from this library.

Reply:
This issue only happened to me in the past but it only affected local development. Did you try to use `wpcomRequest` in Calypso live? If it only works there, we can work on a fix. But this is a can of worms IMO. 

If we really have to use this, then please add context field and pass down from Calypso. The Help Center is already HUGE and importing Calypso stuff destroys tree-shaking.

Reply:
After merging and testing on staging, `wpcomRequest` doesn't seem to work. Similar to local development, I get 403 with missing cookies. 🤔 

I'll figure out how to resolve it.